libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Input/OAuthInput.cs

```diff
@@ -222,8 +222,6 @@ public override async Task<DialogTurnResult> ContinueDialogAsync(DialogContext d
                 {
                     // increase the turnCount as last step
                     dc.State.SetValue(TURN_COUNT_PROPERTY, turnCount + 1);
-                    var prompt = await this.OnRenderPrompt(dc, inputState).ConfigureAwait(false);
-                    await dc.Context.SendActivityAsync(prompt).ConfigureAwait(false);
                     await SendOAuthCardAsync(dc, promptOptions?.Prompt, cancellationToken).ConfigureAwait(false);
```

Comment:
Good catch but this should actually first try to use the RetryPrompt and then fallback to using Prompt if there is no RetryPrompt